src/code/z_player_lib.c

```diff
@@ -1800,49 +1986,396 @@ void Player_DrawZoraShield(PlayState* play, Player* player) {
+                    sPlayerRightHandType = PLAYER_MODELTYPE_RH_CLOSED;
+                } else {
+                    s32 temp_v0_8 = GET_FACE_FROM_JOINTTABLE(player->skelAnime.jointTable) & 0xF00;
+
+                    if (temp_v0_8 != 0) {
+                        temp_v0_8 = (temp_v0_8 >> 8) - 1;
+                        var_v1 = &D_801C0964[temp_v0_8][D_801F59E0];
+                    }
+                }
+
+                *dList = var_v1[D_801F59E4];
+                if (player->skelAnime.animation == &gPlayerAnim_pg_punchB) {
+                    func_80125CE0(player, D_801C0784, pos, rot);
+                }
+            }
+        } else if (limbIndex == PLAYER_LIMB_SHEATH) {
+            Gfx** var_v1_2 = player->sheathDLists;
+
+            if (player->transformation == PLAYER_FORM_HUMAN) {
+                s32 temp_a0 = GET_CUR_EQUIP_VALUE(EQUIP_TYPE_SWORD);
+
+                if (temp_a0 != 0) {
+                    if ((player->sheathType == PLAYER_MODELTYPE_SHEATH_14) ||
+                        (player->sheathType == PLAYER_MODELTYPE_SHEATH_12)) {
+                        var_v1_2 = &gPlayerSheathedSwords[2 * ((temp_a0 - 1) ^ 0)];
+                    } else {
+                        var_v1_2 = &gPlayerSwordSheaths[2 * ((temp_a0 - 1) ^ 0)];
+                    }
+                }
+            }
+
+            *dList = var_v1_2[D_801F59E4];
+        } else if (limbIndex == PLAYER_LIMB_WAIST) {
+            *dList = player->waistDLists[D_801F59E4];
+        } else if (limbIndex == PLAYER_LIMB_HAT) {
+            if (player->transformation == PLAYER_FORM_ZORA) {
+                Matrix_Scale((player->unk_B10[0] * 1) + 1.0f, 1.0f, 1.0f, MTXMODE_APPLY);
+            }
+        }
+    }
+
+    return false;
+}
+
+// Player_OverrideLimbDrawGameplayFirstPerson
+s32 func_801262C8(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* actor) {
+    Player* player = (Player*)actor;
+
+    if (!Player_OverrideLimbDrawGameplayCommon(play, limbIndex, dList, pos, rot, actor)) {
+        if (player->unk_AA5 != 3) {
+            *dList = NULL;
+        } else if (limbIndex == PLAYER_LIMB_LEFT_FOREARM) {
+            *dList = D_801C0294[player->transformation];
+        } else if (limbIndex == PLAYER_LIMB_LEFT_HAND) {
+            *dList = D_801C02A8[player->transformation];
+        } else if (limbIndex == PLAYER_LIMB_RIGHT_SHOULDER) {
+            *dList = D_801C02BC[player->transformation];
+        } else if (limbIndex == PLAYER_LIMB_RIGHT_HAND) {
+            if (Player_IsHoldingHookshot(player)) {
+                *dList = D_801C02E4[player->transformation];
+            } else {
+                *dList = D_801C02D0[player->transformation];
+            }
+        } else {
+            *dList = NULL;
+        }
+    }
+
+    return false;
+}
 
 // unused
 s32 func_801263FC(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx) {
```

Comment:
Could you comment or name that this is a remnant of `Player_OverrideLimbDrawGameplayCrawling` from OoT?